src/pages/blog/index.tsx

```diff
@@ -122,25 +122,30 @@ const BlogPage: FC<{ data: BlogGQL }> = ({ data }) => {
   const { componentType, isDesktop } = useComponentType();
   const developerProfile = useDeveloperProfile();
   const [selectedBlogpost, setSelectedBlogpost] = useState(-1);
+  const dateFormat = 'DD MMMM YYYY';
+
+  if (typeof window !== `undefined` && blogData.blogPost?.length === 0) {
```

Comment:
Is the window condition necessary here? If so please move it to utils and create a method eg. `canUseWindow`

Reply:
It is, will move to utils 😄 
Explaination here => https://github.com/CodersCrew/CodersCard/pull/93#discussion_r504917067